src/renderer/features/governance/model/unlock/confirm-model.ts

```diff
@@ -0,0 +1,173 @@
+import { combine, createStore, sample } from 'effector';
+import { spread } from 'patronum';
+
+import { type Asset, type Chain, type MultisigTxWrapper, type ProxyTxWrapper, type Wallet } from '@shared/core';
+import { networkModel } from '@/entities/network';
+import { transactionService } from '@/entities/transaction';
+import { accountUtils, walletModel, walletUtils } from '@entities/wallet';
+
+const $networkStore = createStore<{ chain: Chain; asset: Asset } | null>(null);
```

Comment:
`networkSelectorModel` already has selected chain and `votingAssetModel` already has selected voting asset